kernel/x86_64/sgemv_n_4.c

```diff
@@ -305,9 +343,6 @@ int CNAME(BLASLONG m, BLASLONG n, BLASLONG dummy1, FLOAT alpha, FLOAT *a, BLASLO
 	BLASLONG lda8 =  lda << 3;
 	FLOAT xbuffer[8],*ybuffer;
 
-        if ( m < 1 ) return(0);
-        if ( n < 1 ) return(0);
```

Comment:
Why this deletion - it looks like a safe optimization to me to do a quick return here if there is no work to be done ?

Reply:
Thanks for review, Martin. Yes, they are useful, not deleted, but revised to be one line at line 297 as below:
  if ( m < 1 || n < 1) return(0);

Reply:
Ah, silly me - sorry.